Week6/app/src/main/java/com/oliverperez/blockbusterchallenge/ui/dashboard/MovieDashboardActivity.kt

```diff
@@ -68,14 +57,17 @@ class MovieDashboardActivity : AppCompatActivity(), MovieGridAdapter.MovieClickL
     }
 
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
-        LoginPrefs.saveUserLoginStatus(false)
-        Toast.makeText(this, "You logged out", Toast.LENGTH_LONG).show()
-        showLogInScreen()
+        if (item.itemId == R.id.menu_item) {
+            LoginPrefs.saveUserLoginStatus(false)
```

Comment:
Rename this method to something that makes more sense. Maybe `isUserLoggedIn()`

Reply:
Also, there's no need to keep LoginPrefs as a singleton. It's better if you create interface-implementation combination and use that instead of an `object`.